crates/cheatnet/src/forking/state.rs

```diff
@@ -248,3 +252,25 @@ impl StateReader for ForkStateReader {
         ))
     }
 }
+
+fn generate_casm(sierra_contract_class: &Value) -> CasmContractClass {
+    let mut temp_sierra_file = Builder::new().tempfile().unwrap();
+    let _ = temp_sierra_file
+        .write(
+            serde_json::to_vec(sierra_contract_class)
+                .unwrap()
+                .as_slice(),
+        )
+        .unwrap();
+
+    let casm = Command::new("universal-sierra-compiler")
+        .args(vec![
+            "--sierra-input-path",
+            temp_sierra_file.path().to_str().unwrap(),
+        ])
+        .output()
+        .context("Error while compiling Sierra of forked contract")
+        .unwrap()
+        .stdout;
+    serde_json::from_slice(&casm).unwrap()
+}
```

Comment:
Isn't this something that we'll want to reuse later on?

Reply:
Do you mean that we should cache that?

Reply:
I mean as for the code, we will want to reuse it so it should be somewhere else

Reply:
You're right. We do use the code running `universal-sierra-compiler` here and in the scarb-api.
I wonder what is the best place to put it in? Should I create a crate `universal-sierra-compiler-api`? Or do you maybe have a better idea?

Reply:
I'm currently working on the `universal-sierra-compiler-api` crate. Will add it in the separate PR